apps/modernization-ui/src/apps/search/patient/PatientCriteria/BasicInformation.tsx

```diff
@@ -0,0 +1,195 @@
+                        onBlur={onBlur}
+                        onChange={onChange}
+                        type="text"
+                        label="Last name"
+                        name={name}
+                        defaultValue={value}
+                        htmlFor={name}
+                        id={name}
+                        error={error?.message}
+                    />
+                )}
+            />
+            <Controller
+                control={control}
+                name="firstName"
+                rules={validNameRule}
+                render={({ field: { onBlur, onChange, value, name }, fieldState: { error } }) => (
+                    <Input
+                        onBlur={onBlur}
+                        onChange={onChange}
+                        defaultValue={value}
+                        type="text"
+                        label="First name"
+                        name={name}
+                        htmlFor={name}
+                        id={name}
+                        error={error?.message}
+                    />
+                )}
+            />
+            <Controller
+                control={control}
+                name="dateOfBirth"
+                render={({ field: { onChange, value, name } }) => (
+                    <DatePickerInput
+                        defaultValue={value}
+                        onChange={onChange}
+                        name={name}
+                        disableFutureDates
+                        label="Date of birth"
+                    />
+                )}
+            />
+            <Controller
+                control={control}
+                name="gender"
+                render={({ field: { onChange, value, name } }) => (
+                    <SelectInput
+                        defaultValue={asValue(value)}
+                        onChange={onChange}
+                        name={name}
+                        htmlFor={name}
+                        label="Sex"
+                        id={name}
+                        options={[
+                            { name: 'Male', value: 'M' },
+                            { name: 'Female', value: 'F' },
+                            { name: 'Other', value: 'U' }
+                        ]}
+                    />
+                )}
+            />
+            <Controller
+                control={control}
+                name="id"
+                render={({ field: { onChange, value, name } }) => (
+                    <Input
+                        onChange={onChange}
+                        defaultValue={value}
+                        type="text"
+                        label="Patient ID"
+                        name={name}
+                        htmlFor={name}
+                        id={name}
+                    />
+                )}
+            />
+            <FormGroup error={!!control._formState.errors.status} className={styles.checkboxes}>
```

Comment:
Issue: do not access values in a control directly. https://www.react-hook-form.com/api/useform/control/

```
Important: do not access any of the properties inside this object directly. It's for internal usage only.
```